lib/knockout.autocomplete.js

```diff
@@ -382,11 +396,15 @@
                         if (suggestions().length) {
                             this.state(true);
                             updateVisibilityOption(true);
+                            $dropdown.on('mouseenter', mouseEnterDropdown)
+                                     .on('mouseleave', mouseLeaveDropdown);
```

Comment:
I played around with this a bit, and it seems like I can provoke getting the `mouseenter`/`mouseleave` handlers attached multiple times without them being removed in the `else` below. I logged `on` when attaching the events and `off` when detaching them, then adding a test that triggered a bunch of `change` events in the input field.

I wanted to provide a test that demonstrated it, but got distracted with something else, so I thought I'd let you know anyway in case you have an idea yourself :)